tools/check-cla

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+author=${1/\/cloud-init/}
+if [ `grep -c ${author} tools/.lp-to-git-user` = 0 ]; then
+    cat <<EOF
+______________________________________________________________
+
+Hello $author,
+
+Thank you for your contribution to cloud-init.
+
+In order to accept this pull request, you need to have signed
+the Contributor License Agreement (CLA). Please ensure that
+your have signed the CLA by following our hacking guide at:
+https://cloudinit.readthedocs.io/en/latest/topics/hacking.html
+______________________________________________________________
```

Comment:
I wonder if we should leave a breadcrumb here for people who have signed the CLA but not transferred it to GH. They may think "I've signed the CLA, this message is just wrong" without clicking through to the HACKING doc at all. Maybe:

> (If you have previously signed the CLA, you will need to transfer your signature to GitHub. The hacking guide includes instructions on how to do this.)